test/controllers/commitments_controller_test.rb

```diff
@@ -44,6 +44,8 @@ class CommitmentsControllerTest < ActionDispatch::IntegrationTest
     sign_in users(:user_1)
```

Comment:
Getting an error on this test

<img width="1240" alt="Screenshot 2022-05-20 at 15 40 10" src="https://user-images.githubusercontent.com/40244233/169551854-23388b01-a4dd-4c05-9142-52a295a0fb89.png">